### PR TITLE
JP : Changes

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Delete all Animax channels : No stream works as of today.

Mention that this specific Tokyo MX1 stream is not 24/7 (sort of official stream, copyrighted content doesn't often get into stream).

### CODE REVIEW

File: channels/jp.m3u [hunk 1]
```diff
@@ -1,18 +1,4 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="AnimaxAsia.jp" tvg-name="Animax Asia" tvg-country="JP" tvg-language="Japanese" tvg-logo="" group-title="",Animax Asia (360p)
-https://tvonlineindonesia.xyz:443/m3u8/usee-animax-People+62.m3u8
-#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese" tvg-logo="" group-title="Movies",Animax Japan (720p)
-http://180.92.158.208:9981/stream/channelid/806982131
-#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese" tvg-logo="" group-title="",Animax Japan (720p)
-https://cdn.jpth10.jpnettv.live/jptv/janimax_720/index.m3u8
-#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Indonesian" tvg-logo="" group-title="",Animax Japan (Indonesian Subs)
-http://210.210.155.35/dr9445/h/h02/01.m3u8
```
Comment (freearhey) [inline channels/jp.m3u:9]: Screenshot 2021-05-07 at 22 11 25

File: channels/jp.m3u [hunk 2]
```diff
@@ -1,18 +1,4 @@
...
-#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Indonesian" tvg-logo="" group-title="",Animax Japan (Indonesian Subs)
-http://210.210.155.35/dr9445/h/h144/01.m3u8
```
Comment (freearhey) [inline channels/jp.m3u:11]: Screenshot 2021-05-07 at 22 12 19

File: channels/jp.m3u [hunk 3]
```diff
@@ -1,18 +1,4 @@
...
-#EXTINF:-1 tvg-id="AnimaxJapan.jp" tvg-name="Animax Japan" tvg-country="JP" tvg-language="Japanese;Vietnamese" tvg-logo="" group-title="",Animax Japan (Vietnamese Subs) (720p)
-https://livecdn.fptplay.net/hda3/animaxport_2000.stream/.m3u8
```
Comment (freearhey) [inline channels/jp.m3u:13]: Screenshot 2021-05-07 at 22 13 11

**Review by freearhey — CHANGES_REQUESTED** [review]
Please the marked links again.

**Comment by LaneSh4d0w** [comment]
Done.

**Review by freearhey — APPROVED** [review]

**Pull request merged**